spec/features/cases/filters/assigned_business_unit_filter_spec.rb

```diff
@@ -70,8 +70,8 @@
     end
 
   end
-  
-  
+
+
```

Comment:
being super fussy but can we get rid of these empty lines

Reply:
Will do.